x/epochs/abci.go

```diff
@@ -65,6 +69,8 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 		}
 
 		k.AfterEpochEnd(ctx, epochInfo.Identifier, epochInfo.CurrentEpoch)
+		k.BeforeEpochStart(ctx, epochInfo.Identifier, epochInfo.CurrentEpoch)
```

Comment:
I'm confused why we have this logic in the end blocker at all. I thought we're going with only having begin blocker logic, no end blocker logic. https://github.com/osmosis-labs/osmosis/issues/149#issuecomment-889626135

(Or is there a reason to move these things to the end blocker)